spec/models/assignment_spec.rb

```diff
@@ -1026,7 +1025,12 @@
           result = s.get_latest_result
           result.total_mark = total_mark
           result.marking_state = Result::MARKING_STATES[:complete]
-          result.save
+          @assignment.rubric_criteria.each do |cri|
+            result.marks.create!(markable_id: cri.id,
+                                 markable_type: RubricCriterion,
+                                 mark: (total_mark * 4.0 /  20).round)
```

Comment:
Unnecessary spacing detected.<br>Operator `/` should be surrounded by a single space.